File: WebAPK/WebAPK/Models/Product.cs
using System;
using System.Collections.Generic;

namespace WebAPK.Models;

public partial class Product
{
    public int Id { get; set; }

    public string Name { get; set; } = null!;

    public double Price { get; set; }

    public int Amount { get; set; }

    public string? Description { get; set; }

    public byte[]? Image { get; set; }

    public int SellerId { get; set; }


    public virtual User Seller { get; set; } = null!;
}

File: Frontend/react-app/src/models/product.js
export class Product {
    constructor(obj) {
      this.id = obj.id;
      this.name = obj.name;
      this.price = obj.price;
      this.amount = obj.amount;
      this.description = obj.description;
      this.image = obj.image;
      this.imageFile = obj.imageFile;
      this.sellerId = obj.sellerId;
      this.seller = new Seller(obj.seller);
    }
  }